tests/cbmc/proofs/Makefile.common

```diff
@@ -60,9 +60,14 @@ LOGDIR = $(PROOFDIR)/logs
 
 ################################################################
 # Useful macros for running commands
+
 #1: command, 2: flags, 3: log file
 DO_AND_LOG_COMMAND = $(1) $(2) 2>&1 | tee $(3); exit $${PIPESTATUS[0]}
 
+#1: command, 2: flags, 3: log file
+DO_AND_LOG_COMMAND_EAT_ERRORS = $(1) $(2) 2>&1 | tee $(3)
```

Comment:
Why are we getting rid of errors here?

Reply:
CBMC raises an error-code when it finds an assertion violation, which means that the build stops and we don't ever get to see the trace.  I will add a comment explaining this.  

Alternatively, the best way would be to check for error code 10, which CBMC uses for assertion violations, continue on that, and stop the build for everything else.

Reply:
Updated the makefile to do the right thing.